src/Components/utils.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MarkdownEditor } from './utils';
+
+describe('MarkDownEditor component', () => {
```

Comment:
Markdown (no captial D) - it doesn't affect how the test runs, just for consistency!